com.microsoft.mrtk.core/Subsystems/Speech/SpeechRecognitionSubsystemDescriptor.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Subsystems
+{
+    /// <summary>
+    /// Encapsulates the parameters for creating a new <see cref="SpeechRecognitionSubsystemDescriptor"/>.
+    /// </summary>
+    public class SpeechRecognitionSubsystemCinfo : MRTKSubsystemCinfo
+    {
+        /// <summary>
+        /// Specifies whether the <c>SpeechRecognitionSubsystem</c> is cloud based.
+        /// </summary>
+        public bool IsCloudBased { get; set; }
```

Comment:
Why would a dev need to know this? Is it to have them expect longer latency?

Reply:
Yeah cloud almost always has a longer latency also they can make decisions based on connection availability.